### PR TITLE
[SBS-17] UI : Cart Screen

### PR DESCRIPTION
**Pull request by leanhdung0505**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

Screenshot_1715138418
Screenshot_1715138414

### CODE REVIEW

File: app/navigators/DemoNavigator.tsx [hunk 1]
```diff
@@ -1,27 +1,23 @@
...
-  DemoShowroom: { queryIndex?: string; itemIndex?: string };
-  DemoDebug: undefined;
-  DemoPodcastList: undefined;
-  DemoCreateProduct: undefined;
-};
+  DemoCommunity: undefined
+  DemoShowroom: { queryIndex?: string; itemIndex?: string }
+  DemoDebug: undefined
+  DemoPodcastList: undefined
+  DemoCartList: undefined
```
Comment (huynamboz) [inline app/navigators/DemoNavigator.tsx]: Sao không thêm param mà lại sửa param DemoCreateProduct, pull code chưa

File: app/data/data.tsx
Comment (huynamboz) [inline app/data/data.tsx:1]: Phần image dùng link ảnh k down ảnh bỏ vô source, sau này cái ảnh lấy từ api mà

File: app/components/CartItem/CartItem.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+                  } } />
+        </View>
+      </View>
+    </View>
+  );
+};
+
+export default CartItem;
+
+const styles = StyleSheet.create({
```
Comment (huynamboz) [inline app/components/CartItem/CartItem.tsx]: Style chuyển sang kiểu ví dụ $container: ViewStyle { ....} cho đúng convention, xem ở những file khác

File: app/navigators/DemoNavigator.tsx (hunk 1 above)
Comment (leanhdung0505) [reply app/navigators/DemoNavigator.tsx]: Có rồi mà t chưa push lên lại

File: app/components/CartItem/CartItem.tsx (hunk 2 above)
Comment (leanhdung0505) [reply app/components/CartItem/CartItem.tsx]: Oke

**Pull request closed without merging**